### DISCUSSION TITLE
Support for Ollama PHP Client and serverside manipulation via PHP with native MySQL

### DISCUSSION THREAD

**Discussion by JennyBatal** [open-webui/open-webui#5450] [Ideas]
Based on the following ollama/php library, https://github.com/jdecool/ollama-php-client, I think it is a powerful feature if we add support for native PHP and MySQL for developer level extensibility purposes, whether they will backed in docker or via the backend interface or port mapping via nodejs.

This will draw ideas on rapid  tool development with server-side applications and API calls to custom programs such as for campuses and backoffice or business servers at home or from a datacenter perspective purposefully designed to extend our great OWUI.

Thanks

**Comment by oliverbob** [comment]
I would be happy to see this supported and implemented. Its really a great way to extend the capabilities of the web ui.

**Comment by justinh-rahb** [comment]
No. Just no. No further explanation need be given.

**Comment by oliverbob** [comment]
Why can there be no explanation?

**Comment by nptrainor** [comment]
This is a great idea.
